Formula/quicktype.rb

```diff
@@ -0,0 +1,19 @@
+require "language/node"
+
+class Quicktype < Formula
+  desc "Generate types and converters from JSON, Schema, and GraphQL"
+  homepage "https://github.com/quicktype/quicktype#readme"
+  url "https://registry.npmjs.org/quicktype/-/quicktype-6.0.0.tgz"
+  sha256 "f615dc2c026a3125039ba4463d8c931a7a771a496366c9aae252512a09a90854"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    system "#{bin}/quicktype", "-o", "LatestBlock.cs", "https://blockchain.info/latestblock"
```

Comment:
Can we check that the LastestBlock.cs file has indeed been written? Or better, match some known content in it?

Reply:
May I install test-only dependencies? quicktype generates code, so if I could install a compiler, I can run a comprehensive test.

Reply:
No test-only deps. But a good intermediate would be to grep for some known pattern in the generated code, for example.

Reply:
Done.